website/content/docs/release-notes/nomad/v1-10-x.mdx

```diff
@@ -0,0 +1,147 @@
+---
+layout: docs
+page_title: Nomad v1.10.x release notes
+description: >-
+  HashiCorp Nomad version 1.10.x release notes. New features include dynamic host volumes, OIDC. Updates to the Container Storage Interface (CSI) volume delete and volume status commands. CSI volume and plugin events added to the event stream. Added UI URL hints to the end of common CLI commands and a -ui flag to auto-open them in a browser.
+---
+
+# Nomad 1.10.x release notes
+
+We are pleased to announce the following Nomad updates.
+
+## Release highlights
+
+#### Dynamic host volumes
+
+The dynamic host volumes feature brings a persistent storage option to your
+workload allocations.
+
+Nomad dynamic host volumes manage storage for stateful workloads without
+requiring a restart of the Nomad nodes to apply configuration changes. They
+provide a workload-agnostic way to specify resources and work with the `exec`,
```

Comment:
This "workload-agnostic way to specify resources" clause feels like filler... I don't think means anything in particular.